II-9865-develop-Python-agent-for-Google-metric-monitoring-API

atriekak opened the pull request.

zhangzinan reviewed (approved):
For new agent, It is better to follow the README of servicenow agent. This agent has new workflow.
https://github.com/insightfinder/InsightAgent/tree/master/servicenow

atriekak commented:
> For new agent, It is better to follow the README of servicenow agent. This agent has new workflow. https://github.com/insightfinder/InsightAgent/tree/master/servicenow

Addressed the comment. Have added to the README file. My workflow does not include .sh files following a discussion that it was preferable to add directions to set up the agent instead.
